### PR TITLE
Community update monitoring and BU FAQs and remove community links fro…

### PR DESCRIPTION
**Pull request by catlook**
…m other article.

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7d1f18d243/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f07c8f223c/how-to/).

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 1]
```diff
@@ -192,7 +212,69 @@ addressed.
...
+availability of a computer (node) on a network. If the node responds, the Ping
+utility also measures how long it take for a small packet of information to
+make a round trip from your computer to that remote system. This check monitors
+the general responsiveness of your server on the network and alerts you if it
+fails to respond.
+
+#### What is an agent check?
+
+You need to install a monitoring agent on your cloud server to use agent checks.
+If you have a cloud account with a managed service level, the build process
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: "managed service level agreement"?

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 2]
```diff
@@ -192,7 +212,69 @@ addressed.
...
+installs the monitoring agent for you as part of the build process. If you have
+an infrastructure account, you need to install the agent manually.
+
+After the agent is installed, you can see current and historical performance
+information about a cloud server from it's detail screen. Agent checks enable
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: "...from its **Details** screen in the Cloud Control Panel."

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 3]
```diff
@@ -192,7 +212,69 @@ addressed.
...
+you to set specific thresholds that trigger notifications.
+
+The following list briefly describes the available Agent Checks:
+
+- **Memory Check**: Your server has a finite amount of memory. Running low on
+memory negatively affects the performance of your entire server and might cause
+it to be unresponsive. This check alerts you when your cloud server’s memory
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: Suggest "...become unresponsive."

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 4]
```diff
@@ -192,7 +212,69 @@ addressed.
...
+utilization surpasses 80%, but that value can be changed to meet your needs.
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: Suggest "...but you can change that value to meet your needs."

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 5]
```diff
@@ -192,7 +212,69 @@ addressed.
...
+
+- **CPU Check**: Like memory, your server is useless if it runs out of CPU. This
+check returns a warning for 90% CPU used and a critical warning for 95% CPU used.
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: Suggest "By default, this check returns a warning when 90% of CPU is used and a critical warning when 95% of CPU is used. You can configure these thresholds to suit your needs."

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 6]
```diff
@@ -211,13 +293,13 @@ them.
...
-and built for the cloud. Rackspace Monitoring also provides geographically redundant
-checks, which is generally difficult to get with any solution. Customers can leverage our
-big data center footprint, incredible scalability, and our continuous release feature.
-Future improvements to the service are released as they become functional, so there is
-no need to wait for an upgrade package, and no need for downtime.
+Rackspace Monitoring is intended to replace these types of tools. Although Rackspace
+Monitoring doesn’t offer all the features of Nagios&reg;, it is hosted as a service,
+API driven, and built for the cloud. Rackspace Monitoring also provides geographically
+redundant checks, which is generally difficult to get with any solution. Customers
+can leverage our big data center footprint, incredible scalability, and our continuous
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: Suggest "...our large..."

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 7]
```diff
@@ -231,7 +313,7 @@ placed into spam folders.
 #### Why separate out the concepts of checks and alarms?
 
 To build a state-of-the-art monitoring platform, data collection should be separate from
-the thresholds. On the CLI/API level, this is a more complex user experience, but it
+the thresholds. On the CLI and API level, this is a more complex user experience, but it
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: Suggest "At the CLI..."

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 8]
```diff
@@ -244,16 +326,18 @@ Rackspace Monitoring Technical Reference.
 
 #### Is there a set of command line tools to use with Rackspace Monitoring?
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: Suggest "...command-line..."

File: content/cloud-backup/cloud-backup-faq.md [hunk 9]
```diff
@@ -128,20 +136,30 @@ data. An added benefit is that this capability enables you to retrieve
 previous versions of your files (up to the limits specified in your data
 retention settings).
 
-See the section on [Scheduling and Deduplication](/how-to/rackspace-cloud-backup-backup-actions) in the list of Cloud Backup actions.
+See the section on [Scheduling and Deduplication](/how-to/rackspace-cloud-backup-backup-actions)
+in the list of Cloud Backup actions.
 
-#### Can I use Rackspace Cloud Backup on my server?
+#### Can I use  Cloud Backup on my server?
```
[the remark this answers is not in the document]
Comment (catlook) [reply content/cloud-backup/cloud-backup-faq.md]: yup!

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 10]
```diff
@@ -61,19 +74,24 @@ Rackspace Monitoring is an API-based system. You can access it using the and [st
 
 #### How will I receive an alert when my website is down?
 
-The service currently supports email, SMS, PagerDuty, VictorOps, and webhook
-notifications. For more information, see the [Notification types](https://developer.rackspace.com/docs/rackspace-monitoring/v1/api-reference/notification-type-operations/)section in the Rackspace Monitoring API Reference.
+The service currently supports email, SHort Message Service(SMS), PagerDuty,
```
[the remark this answers is not in the document]
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: yup

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 11]
```diff
@@ -52,7 +56,16 @@ section in the Rackspace Monitoring Getting Started Guide.
 
 #### How do I configure my monitoring system?
 
-Rackspace Monitoring is an API-based system. You can access it using the and [start creating monitoring checks](/how-to/available-checks-for-rackspace-monitoring/) by using the following methods:
+Cloud Monitoring is used by configuring one or more checks that monitor the
```
[the remark this answers is not in the document]
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: yup

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 12]
```diff
@@ -192,7 +212,69 @@ addressed.
...
+points on the Internet.
+
+The following list briefly describes the available remote service checks:
+
+- **HTTP Check (Website)**: This check monitors the availability of your website
+either by URL or by IP address and alerts you if the site becomes unavailable
+for more than 30 seconds.
+
+- **TCP Check (Port)**: This check monitors the response from a specific port
+on your server, to determine if the process that is bound to that port is running.
```
[the remark this answers is not in the document]
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: yup

File: content/rackspace-monitoring/rackspace-monitoring-faq.md (hunk 1 above)
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: could go either way, but i'll go with yours

File: content/rackspace-monitoring/rackspace-monitoring-faq.md (hunk 2 above)
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: yup

File: content/rackspace-monitoring/rackspace-monitoring-faq.md (hunk 3 above)
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: keeping original in this case

File: content/rackspace-monitoring/rackspace-monitoring-faq.md (hunk 4 above)
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: yup

File: content/rackspace-monitoring/rackspace-monitoring-faq.md (hunk 5 above)
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: lol...you didn't like the useless thing?  alright...

File: content/rackspace-monitoring/rackspace-monitoring-faq.md (hunk 6 above)
Comment (catlook) [reply content/rackspace-monitoring/rackspace-monitoring-faq.md]: I suppose.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e71ac41d62/how-to/)

File: content/rackspace-monitoring/rackspace-monitoring-faq.md [hunk 13]
```diff
@@ -61,19 +74,24 @@ Rackspace Monitoring is an API-based system. You can access it using the and [st
 
 #### How will I receive an alert when my website is down?
 
-The service currently supports email, SMS, PagerDuty, VictorOps, and webhook
-notifications. For more information, see the [Notification types](https://developer.rackspace.com/docs/rackspace-monitoring/v1/api-reference/notification-type-operations/)section in the Rackspace Monitoring API Reference.
+The service currently supports email, SHort Message Service (SMS), PagerDuty,
```
Comment (RackerKate) [inline content/rackspace-monitoring/rackspace-monitoring-faq.md]: "Short"

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2ec13090e8/how-to/)

**Pull request merged**